README.md

```diff
@@ -137,7 +145,9 @@ Use [`framework/bin/defects4j`](http://defects4j.org/html_doc/defects4j.html) to
 | [mutation](http://defects4j.org/html_doc/d4j/d4j-mutation.html)           | Run mutation analysis on a buggy or a fixed project version                                       |
 | [coverage](http://defects4j.org/html_doc/d4j/d4j-coverage.html)           | Run code coverage analysis on a buggy or a fixed project version                                  |
 | [monitor.test](http://defects4j.org/html_doc/d4j/d4j-monitor.test.html)   | Monitor the class loader during the execution of a single test or a test suite                    |
+| [bids](http://defects4j.org/html_doc/d4j/d4j-bids.html)                   | Print the list of active bug IDs for a specific project                                           |
```

Comment:
Print the list of active or deprecated bug IDs for a specific project

Reply:
Fixed in latest commit